### DISCUSSION TITLE
Explorerpatcher not working today

### DISCUSSION THREAD

**Discussion by PocketDeucesNJ** [valinet/ExplorerPatcher#1793] [Q&A]
Maybe something was broken in the latest Windows 11 update?  My taskbar is back to standard Windows.  I tried uninstalling and reinstalling EP, but no dice.

**Comment by smokingtux** [comment]
Same for me.

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Reply by smokingtux** [reply]
Explorer Patcher: 22621.1992.56.1

OS Build:
23H2
22631.2129
Windows Feature Experience Pack 1000.22669.1000.0

It seems like it's started, but the old Taskbar (from Windows 10) can't be loaded. 
It's started after an update.
See the screenshot, which update was done below.
https://imgur.com/a/Pvsnvg2

**Comment by pyrates999** [comment]
That is an insider build: https://blogs.windows.com/windows-insider/2023/08/02/announcing-windows-11-insider-preview-build-22621-2129-and-22631-2129/

Set Explorer patcher to not download symbols since insider builds have them included: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

You may need to wait for this to get fixed or added as part of a regular update before it may get fixed though.

Is there any technical reason why you are running an insider build?

You can also try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

After, you can check what happens when using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.  Then recreate the possible bug you found and post the output from the console.

**Comment by sanjaysrikonda66** [comment]
I'm running the following (yes, I have the dev channel installed, no, I didn't realize it until now):

Edition	Windows 11 Home Insider Preview
Version	22H2
Installed on	‎8/‎5/‎2023
OS build	23511.1000
Experience	Windows Feature Experience Pack 1000.23511.1000.0

What's happening since the latest release is the following:

1. Installed/uninstalled explorerpatcher
2. deleted registry key cited above
3. installed explorerpatcher again
4. No indication about symbols being downloaded at all, nothing comes up that it doesn't have them
5. The items for moving the taskbar from the default bottom to anywhere else, doesn't even register at all

I guess I wait for a patch.
Just reporting what I'm seeing.

**Comment by PocketDeucesNJ** [comment]
Found another solution.  Unenroll from the Windows Insider program.  Then, go into Windows Update and uninstall the update from today, 8/7.

**Reply by smokingtux** [reply]
Thanks, that fixed it also for me.

**Reply by vippium** [reply]
Bcoz something MS has changed which is not allowing EP to use classic taskbar after that beta build. Waiting for dev to solve this issue.

**Comment by zaadstra** [comment]
EP broken last night, W11 22H2 22621.2134.  Updates seem irreversable:

![image](https://github.com/valinet/ExplorerPatcher/assets/29779266/e3393566-915a-4c7d-8ba3-bc504b1ee629)

"Quality updates" :-(

**Reply by pyrates999** [reply]
You gotta wait for it to be fixed then.

**Reply by zaadstra** [reply]
Just received a symbol update and restarted explorer.  Stuff is back to normal.
Great work, thanks!

**Reply by ivanlan9** [reply]
zaadstra, are you using an insider build?

**Reply by zaadstra** [reply]
No (presumably, I'm using a W11 factory restore).  Versions are in this or the other thread.
Insider Program is switched off (just checked).

**Comment by PocketDeucesNJ** [comment]
How do you get the symbol update if explorer patcher isnt accessible?

**Reply by zaadstra** [reply]
I was away so PC was shut down for a while. After bootup I saw a tray popup "symbols downloaded - restart Explorer (or EP, don't know)".
Startmenu - "ExplorerPatcher" (typed in and run) - click 'Restart Explorer" et voila!

**Reply by pyrates999** [reply]
symbols are downloaded in the background.  You can also check to see if symbols are trying to be downloaded by opening up the properties application for explorer patcher, go to advanced, and click on enable console.

**Reply by zaadstra** [reply]
Presumably updated symbols need to be available then? I understand that updated symbols did the repair trick here.
Fixing the issue in just one day is awesome.

**Reply by sanjaysrikonda66** [reply]
The symbols version in console reads as:
Local version obtained from hash is 22622.1992.56.1.

**Comment by PocketDeucesNJ** [comment]
The issue is that it looks like explorer patcher isn't running at all for
me. I don't know how to get to the properties or any other screens for it.

**Reply by sanjaysrikonda66** [reply]
Yes, same issue. Some of the items in the app work, others don't. Restart explorer works until it crashes and tells me it crashed, other items don't even register such as moving the nav bar to the top. 

I've unenrolled from the insider preview but I guess I'll wait and see what happens.

**Reply by zaadstra** [reply]
Startmenu - "ExplorerPatcher" (type it in and run it)

**Comment by aseem-thakkar** [comment]
Same for me. The task switcher is broken, windows switcher(alt+tab/wind+tab) are broken. tried installing older releases but nothing works.

**Comment by pixelloop** [comment]
The entire app crashes explorer for me on latest build of Windows and eventually defaults to Windows 11 when I'm trying to set it to Windows 10.

**Reply by bal3wolf** [reply]
yea i have 1 system crashing like crazing but another is working fine with patcher and new updates.

**Reply by pixelloop** [reply]
I upgraded mine once I noticed the issue after Windows updated, but the issue persists. Is there a separate patcher I'm missing?

**Comment by PocketDeucesNJ** [comment]
I've been having strange issues on two separate PCs both with explorer
patcher installed. It seems that the desktop either just goes to black or
flickers after or the windows update installs. Do we need to wait until
explorer patcher is fixed before reinstalling?

**Reply by sanjaysrikonda66** [reply]
Holy moly, I didn't know what to attribute that behavior to. Now I get where it's coming from. I think I'll just remove explorerpatcher for the time being.

**Reply by pixelloop** [reply]
Same thing I'm experiencing.

**Comment by ivanlan9** [comment]
I can't run EP at all:  when it runs, FileExplorer crashes every ten seconds, and after a few of those, EP pops up and says it's disabled, but FE keeps crashing and I am unable to run or uninstall & reinstall EP at all. If I'm missing something I'd be DELIGHTED to be wrong.

This started only today, about three hours ago, when I installed the latest Windows 11 cumulative update for 22H2 which includes KB5029263.  Update also broke WindHawk and OpenShell.

**Comment by PocketDeucesNJ** [comment]
I think it's probably limited to insider builds. I've unenrolled from the
insider program but we have to wait for the next official windows update
override it if you don't want to reinstall windows from scratch.

**Comment by ivanlan9** [comment]
I'm not using an insider build.  I'd definitely have said if I were using one.

Haven't used them since Windows 7. ...

**Reply by pixelloop** [reply]
Same and I'm experiencing the issue too.

**Comment by pyrates999** [comment]
What is this discussion about again?  Saying it isn't working doesn't tell anyone anything.

**Comment by ivanlan9** [comment]
I'm not sure about others, but for me, after the W11 update yesterday, FileExplorer crashes every few seconds, and every 10-15 seconds ExplorerPatcher notices and says it's disabling itself to give time to deal with the problem.  But you can't, because FileExplorer KEEPS crashing. Once you're able to uninstall ExplorerPatcher, you can reboot and FileExplorer stops crashing.

**Comment by Udimi** [comment]
Stopped working after KB5029263 update. I have deleted it and it started working again, so it's clear that update is the problem

**Reply by matiasggg** [reply]
Just uninstalled KB5029263, rebooted and it started working again. Thanks.

**Comment by ivanlan9** [comment]
I was able to uninstall the KB5029263 part of the update, but it made no difference,  The "Cumulative Update" part of the update is not allowed to be uninstalled, and I assume that's the part that's causing the problem.

But uninstalling that part of the update allowed Windhawk and Fences to run without crashing.

**Comment by PocketDeucesNJ** [comment]
Unfortunately these updates keep reinstalling themselves... We need a
better fix.

**Reply by pyrates999** [reply]
You can pause updates for 30 days.

**Comment by bal3wolf** [comment]
not the best answer but i broke down and bought startallback for 5 bucks to get my windows the way i want it.  i like patcher but till its fixed i cant use my system way it was.

**Comment by sanjaysrikonda66** [comment]
Wow, I just downloaded and used this. No issues at all. Thanks.

**Comment by PocketDeucesNJ** [comment]
Good luck with StartAllBack – I had similar issues with that and had to do a system restore to get to my desktop.

**Reply by bal3wolf** [reply]
so far its working great but yea anytime your using software like startallback, start11, and explorer patcher it can toast your OS but iv fixed much worse in the past.
